README-for-review.md

```diff
@@ -0,0 +1,384 @@
+
+![Jayme's Architecture In A Nutshell](https://raw.githubusercontent.com/inaka/Jayme/master/Assets/architecture-diagram-1.png)
+
+## Migration Guides
+
+- [Jayme 2.0 Migration Guide](https://github.com/inaka/Jayme/blob/master/Documentation/Jayme%202.0%20Migration%20Guide.md)
+
+##Features
+
+- **Protocol-Oriented**
+  - Jayme has been built following the concepts of [protocol-oriented programming](https://developer.apple.com/videos/play/wwdc2015/408/), encouraging composition over inheritance whenever possible.
+- **Generics**
+  - In order to provide high flexibility, generics and associated types are present almost everywhere in the library.
+- **Error Handling**
+  - Jayme comes with a default list of discrete errors, which are defined in an enumeration named `JaymeError`.
+  - Any `case` in `JaymeError` turns out to be useful from the `ViewController` layer point of view. Cases represent possible different and meaningful UI scenarios. For instance: a `5xx` server status code is represented by a `case ServerError(statusCode: Int)`.
+  - If you need different error definitions, Jayme allows you to use your own Error types, with the aid of associated types.
+- **Futures / Results**
+  - From experience, we've found out that the [Future Pattern](https://realm.io/news/swift-summit-javier-soto-futures/) is a very convenient way for layouting asynchronous code. In consequence, we decided to develop Jayme around that pattern.
+  - `Future` and `Result` are two key structures in the library. You need to be familiar with them, since they are exposed on its public interfaces.
+- **Logs**
+  - Jayme includes a practical logging mechanism that can be quickly enabled or disabled. It also allows you to set a custom logging function to use, which results very convenient if your project uses third party logging libraries, like [CocoaLumberjack](https://github.com/CocoaLumberjack/CocoaLumberjack).
+- **Unit Tests**
+  - Jayme is 100% unit-tested.
+  - Unit tests are easy to implement, and encouraged, for testing your repositories business logic and your entities parsing.
+- **No Dependencies**
+  - Jayme does not leverage any external dependency.
+  - However, you can integrate JSON parsing libraries (like [SwiftyJSON](https://github.com/SwiftyJSON/SwiftyJSON)) and plug them in easily within `DictionaryInitializable` and `DictionaryRepresentable` protocol functions implementations in your entities.
+
+
+
+
+
+## Architecture
+
+Jayme works around the **Repository Pattern**. There are some key concepts that you need to become familiar with before using the library. These concepts can be outlined differently depending on how the pattern is implemented, as there are several ways of working with it.
+
+Below are brief descriptions of these concepts, based on how Jayme in particular implements the pattern:
```

Comment:
Besides the first part, that 'in particular' is the one that doesn't fit well in the sentence. We can either place it at the end of the sentence, as long as it keeps the same idea that you wanted to say, or put it between comas and erasing the one in based on: 'Below you'll find some brief descriptions of these concepts based on how Jayme, in particular, implements this pattern: